source/ProcessManager.Components.Abstractions/ProcessManager.Components.Abstractions.csproj

```
@@ -7,7 +7,7 @@
 
   <PropertyGroup>
     <PackageId>Energinet.DataHub.ProcessManager.Components.Abstractions</PackageId>
-    <PackageVersion>1.3.7$(VersionSuffix)</PackageVersion>
+    <PackageVersion>1.4.0$(VersionSuffix)</PackageVersion>
```

See my comment for the ReleaseNotes.md
```suggestion
    <PackageVersion>1.3.8$(VersionSuffix)</PackageVersion>
```